prac_04/repeated_string_printer.py

```diff
@@ -39,6 +41,7 @@ def print_repeated_strings(repeated_strings):
 def main():
     print_instructions()
     string_sequence = get_string_sequence()
+    #string_sequence = ["hello", "world", "hello", "world", ".", ""]
     repeated_strings = find_repeated_strings(string_sequence)
     print_repeated_strings(repeated_strings)
 
```

Comment:
Great use of functions presented in the course so far. Maybe add some comments to aid a reviewer reading the program?